Desktop/widgets/jasplistcontrol.h

```diff
@@ -89,7 +89,7 @@ class JASPListControl : public JASPControl
 			const QString&		valueRole()					const			{ return _valueRole;			}
 			bool				containsVariables()			const			{ return _containsVariables;	}
 			bool				containsInteractions()		const			{ return _containsInteractions;	}
-			bool				encodeValue()				const override	{ return containsVariables();	}
+			bool				encodeValue()				const override	{ return containsVariables() || containsInteractions();	}
```

Comment:
Why did you add this?

Reply:
Otherwise interactions between levels with encodable entries wouldn't be encoded. And thus RM ANOVA wouldnt work.

`withinModelTerms` specifically

Reply:
I would have then preferred to overwrite encodeValue() for the specific class...

Reply:
Well, you wrote quite a lot of code, I thought this was the most specific place I could get.

But are there lists where the interactions don't need to be encoded?